base/linalg/dense.jl

```diff
@@ -211,6 +211,25 @@ end
     kron(A, B)
 
 Kronecker tensor product of two vectors or two matrices.
+
+```jldoctest
+julia> A = [1 2; 3 4]
+2×2 Array{Int64,2}:
+ 1  2
+ 3  4
+
+julia> B = [1 1; 1 1]
```

Comment:
Perhaps `[im 1; 1 1]` would be slightly more interesting but similarly easy to grasp?